psi4/driver/procrouting/mcscf/mcscf_solver.py

```diff
@@ -355,7 +402,11 @@ def mcscf_solver(ref_wfn):
 
     # Do we diagonalize the opdm?
     if core.get_option("DETCI", "NAT_ORBS"):
+        core.print_out("  \n   Transforming MOs to the natural basis\n")
```

Comment:
As a note this already has a message from the C side, did you want to add additional text here?